fiat-core/src/main/java/com/netflix/spinnaker/fiat/model/resources/BaseAccessControlled.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2017 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.fiat.model.resources;
+
+import com.netflix.spinnaker.fiat.model.Authorization;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.List;
+
+@Slf4j
+abstract class BaseAccessControlled<R extends Resource.AccessControlled> implements Resource.AccessControlled<R> {
```

Comment:
What's the plan for cutting support for `requiredGroupMembership`? Would you just remove this class and have the classes that used to extend this just implement the `AccessControlled` interface?